xlsx/tests/test.rs

```diff
@@ -334,3 +334,72 @@ fn no_export() {
     }
     fs::remove_dir_all(&dir).unwrap();
 }
+
+// This test verifies whether exporting the
+// merged cells functionality is happening
+// properly or not.
+// it first loads the excell
+// having the merged cell and exports it to
+// anothe xlsx and verifies whether merged
+// cell node is same in both of the xlsx file
+#[test]
+fn test_exporting_merged_cells() {
+    let temp_file_name = "temp_file_test_export_merged_cells.xlsx";
+    let expected_merge_cell_ref = {
+        // loading the xlsx file containing merged cells
+        let example_file_name = "tests/example.xlsx";
+        let mut model = load_from_xlsx(example_file_name, "en", "UTC").unwrap();
+        let expected_merge_cell_ref = model
+            .workbook
+            .worksheets
+            .get(0)
+            .unwrap()
+            .merge_cells
+            .clone();
+        // exporting and saving it in another xlsx
+        model.evaluate();
+        save_to_xlsx(&model, temp_file_name).unwrap();
+        expected_merge_cell_ref
+    };
+    {
+        let mut temp_model = load_from_xlsx(temp_file_name, "en", "UTC").unwrap();
+        {
+            //loading the previous file back and verifying whether
+            //merged cells got exported properly or not
```

Comment:
Please add a space between `//` and `merged ..`. Same in the comment above.

Reply:
okay noted and done